wrapper.go

```diff
@@ -40,3 +40,21 @@ func (w *Wrapper) Set(ctx context.Context, key string, value interface{}, expira
 	cmd = w.client.Set(key, value, expiration)
 	return
 }
+
+func (w *Wrapper) Incr(ctx context.Context, key string) (cmd IntCmd) {
+	var recordCallFunc = recordCall(ctx, "go.redis.incr", w.instanceName)
+	defer func() {
+		recordCallFunc(cmd)
+	}()
+	cmd = w.client.Incr(key)
+	return
+
+}
+
+func (w *Wrapper) Ping() (error, string) {
```

Comment:
I'm trying to ensure this wrapper returns the same types (well interface of that type) as the actual clients function calls. It looks like [Ping](https://godoc.org/gopkg.in/redis.v3#Client.Ping) returns a [StatusCmd](https://godoc.org/gopkg.in/redis.v3#StatusCmd) not `(error, string)`. If I'm not missing something I'd like this `Ping` to return the same.

Reply:
What do you think of supplying a `context.Context` to `Ping` and calling `recordCallFunc` ?

Reply:
I'll update ping to return that StatusCmd. As for accepting the context, I also thought about that originally as well, but that Ping command would happen before a request is received so we'd have to send some other context with it which is also probably fine.

Reply:
Using a `context.Background()` would work if there isn't another context available yet